src/services/imaged-object.ts

```diff
@@ -82,25 +82,18 @@ class ImagedObjectService {
         const body = {
```

Comment:
You should fix the position issue also in line 70, where `position:'1'` should be `position: null`.  Currently we cannot create new artefacts, but that change will fix the problem.